building.md

```diff
@@ -4,7 +4,7 @@ Vita3K uses CMake for its project configuration and generation and should in the
 
 Target OS | Host OS | [Project generator](https://cmake.org/cmake/help/latest/manual/cmake-generators.7.html) (`-G`) | C/C++ Compiler (`-DCMAKE_C_COMPILER` and `-DCMAKE_CXX_COMPILER`)
 --- | --- | --- | ---
-Windows | Windows | Visual Studio / MSBuild | `cl` a.k.a Microsoft Visual C/C++ Compiler (included as a part of the [Build Tools for Visual Studio 2019](https://visualstudio.microsoft.com/downloads/#build-tools-for-visual-studio-2019) and the ` Dekstop development with C++` Visual Studio workload)
+Windows | Windows | Visual Studio / MSBuild | `cl` a.k.a Microsoft Visual C/C++ Compiler (included as a part of the [Build Tools for Visual Studio 2022](https://aka.ms/vs/17/release/vs_BuildTools.exe) and the ` Dekstop development with C++` Visual Studio workload)
```

Comment:
s/Dekstop/Desktop

Reply:
?

Reply:
Spelling.

Reply:
oki, thx fixed, i have no see this typo error